mockintosh/res/management.html

```diff
@@ -131,11 +138,11 @@ <H1 style="text-transform: uppercase; padding: 0.25em 0.5em; font-weight: bold;
             <button class="btn btn-sm btn-primary">Load</button>
             <button class="btn btn-sm btn-success">Apply Changes</button>
             <button class="btn btn-info btn-sm pull-right">Reset Dataset Iterators</button>
-            <pre contenteditable="true" class="border rounded" style="margin-top: 0.25em; padding:0.25em"></pre>
+            <pre contenteditable="true" class="border rounded" style="margin-top: 0.25em; padding:0.25em"><code></code></pre>
         </div>
         <div class="tab-pane fade" id="unhandled" role="tabpanel" aria-labelledby="unhandled-tab">
             <button class="btn btn-sm btn-primary">Refresh</button>
-            <pre></pre>
+            <pre><code></code></pre>
```

Comment:
Do we need `pre` if we have `code`?

Reply:
Yes, `pre` is respecting white-spaces, while `code` shows that the following part is a code snippet.